doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+1. Still in the `psql` shell connected to Materialize, remove the `medium` replica from the source cluster:
+
+    ```sql
+    DROP CLUSTER REPLICA ingest_rds_postgres.r1;
+    ```
+
+1. Add an `xsmall` replica to the source cluster:
+
+    ```sql
+    CREATE CLUSTER REPLICA ingest_rds_postgres.r1 SIZE 'xsmall';
+    ```
+
+1. Use the [`SHOW CLUSTER REPLICAS`](/sql/show-cluster-replicas/) command to check the status of the new replica:
+
+    ```sql
+    SHOW CLUSTER REPLICAS WHERE cluster = 'ingest_rds_postgres';
+    ```
+    <p></p>
+
+    ```noftm
+           cluster       | replica |  size  | ready
+    ---------------------+---------+--------+-------
+     ingest_rds_postgres | r1      | xsmall | t
+    (1 row)
+    ```
+
+1. Going forward, you can use the [`mz_source_statistics`](/sql/system-catalog/mz_internal/#mz_source_statistics) table to verify that your new replica size is sufficient for Materialize to keep up with the change events coming through the replication stream:
+
+    ```sql
+    SELECT
+      SUM(updates_staged) AS updates_staged,
+      SUM(updates_committed) AS updates_committed
+    FROM mz_internal.mz_source_statistics
+    WHERE id = '<source_ID>';
+    ```
+
+    <p></p>
+
+    ```nofmt
+     updates_staged | updates_committed
+    ----------------+-------------------
+                  0 |                 0
+    (1 row)
+    ```
+
+    A large gap between `updates_staged` and `updates_committed` is usually an indication that the source has fallen behind and that you might need to scale up your source.
+
+## Next steps
+
+With Materialize ingesting your PostgreSQL data into durable storage, you can start exploring the data, computing real-time results that stay up-to-date as new data arrives, and serving results in various ways.
+
+- Explore
+  - Use [`SHOW SOURCES`](/sql/show-sources) to view the subsources in Materialize that map to tables from your PostgreSQL publication.
+  - Use [`SELECT`](/sql/select/) to explore these subsources just like you would regular tables in PostgreSQL.
+- Transform
+  - Once you have a query that you want Materialize to answer, use [`CREATE VIEW`](/sql/create-view/) to create an alias for the query and then [`CREATE INDEX`](/sql/create-index/) to have Materialize pre-compute the results in memory for fast retrieval.
+  - One thing to note about indexes is that they exist only in the cluster where they are created. When you want results to be accessible from multiple clusters, use [`CREATE MATERIALIZED VIEW`](/sql/create-materialized-view/) to persist pre-computed results to durable storage that is accessible across clusters.
+- Serve
+  - Use [`SELECT`](/sql/select/) to pull results from Materialize.
+  - Use [`SUBSCRIBE`](/sql/subscribe/) to push changes in results to a PostgreSQL-compatible SQL client or driver.
+  - Use [`CREATE SINK`](/sql/create-sink/) to push results to an external message broker.
```

Comment:
Maybe a "Next Steps" link at the bottom here to point them at our general docs?

Reply:
I don't think linking to the docs home page from within the docs is very helpful; these are all the kinds of next steps one might take. If anything, I'd suggest trimming them down a bit to help users take a focused next action like creating a view over their data, but I digress.

Reply:
Yeah, if we had a landing page for explore, transform, and serve, that's where I'd send people. Hope to have that before too much longer. For now, I'll just trim down somewhat.